index.d.ts

```diff
@@ -37,7 +37,7 @@ export type RTCIceConnectionState =
 
 export type RTCPeerConnectionState = 'new' | 'connecting' | 'connected' | 'disconnected' | 'failed' | 'closed';
 
-export type MediaDeviceKind = 'audioinput' | 'audiooutput' | 'videoinput';
+export type MediaDeviceKind = 'audioinput' | 'audiooutput' | 'videoinput' | 'audio';
```

Comment:
Can we technically get rid of 'audioinput' and 'audiooutput', then?

Reply:
Per our discussion: let's remove the ones we're not using since it affects the customer.

Reply:
Fixed!